custom/lib/User.php

```diff
@@ -7,9 +7,7 @@
 use OpenAPI\Client\Configuration;
 use OpenAPI\Client\Api\UsersApi;
 use OpenAPI\Client\Api\UserDevicesApi;
-use OpenAPI\Client\Model\UserInfo as PassageUser;
-use OpenAPI\Client\Model\UpdateUserRequest as UpdateUserArgs;
-use OpenAPI\Client\Model\CreateUserRequest as CreateUserArgs;
```

Comment:
This wasn't working in the Passage.php file so you created the workaround?

Reply:
that was working, but only within the scope of this file. it didn't actually alias the class names for an external dev